js-core-3/week-2/lesson.md

```diff
@@ -175,6 +195,30 @@ counterFromTen.whenClicked();
 
 Variables specific to a particular instance of a class are defined and referenced using the `this` keyword (e.g. `this.timesClicked`) within that instance.
 
+##### Check-in
+**Task (5 mins):**
+Complete the rest of this code to create an online shopping cart.
+```
+class ShoppingCart {
+// Add your code here
+
+}
+
+let myCart = new ShoppingCart(); // Creates an empty shopping cart
+myCart.addItem();
+```
+
+The output of running your code should be:
+
+       You have 1 item in your cart
```

Comment:
Oo great exercise idea! I wonder if it's a bit similar to the Counter example code. How about something like this (or is it too tricky?):

```
class ShoppingCart {
// Add your code here
}
let myCart = new ShoppingCart(); // Creates an empty shopping cart
myCart.addItem("toilet roll");
myCart.addItem("pasta");
myCart.addItem("eggs");

myCart.listItems();
// "Your shopping cart has 3 items: toilet roll, pasta, eggs"
```

Reply:
I'm don't know if all the students would be able to complete this in 2-3 minutes. The more advanced students perhaps, but the weaker students I'm not so sure.

We can use your example, but I'd suggest putting it into the homework rather than during the lesson and we find a different way to quickly assess understanding for the lesson.